### ISSUE TITLE
There are no searchers available in ExamineManager 

### ISSUE THREAD

**Issue by truongtphat** [umbraco/Umbraco-CMS#14749]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

12.0.0

### Bug summary

Hi everyone, 
I followed the [documentation](https://docs.umbraco.com/umbraco-cms/reference/searching/examine/examine-manager) to implement Examine search so I got the results that there aren't any searchers available. Here is my debugging
![image](https://github.com/umbraco/Umbraco-CMS/assets/28688681/23346029-7197-4f4b-8611-00bec0fbc4c9)

Then I read your source code and found an API to get all searchers and got the result as below
![image](https://github.com/umbraco/Umbraco-CMS/assets/28688681/c336dded-0099-4af0-822c-debc5a57c4a5)

### Steps to reproduce

- Create new a Umbraco project
- Follow the [documentation](https://docs.umbraco.com/umbraco-cms/reference/searching/examine/examine-manager) 
- Execute API yourdomain/umbraco/backoffice/umbracoapi/examinemanagement/GetSearcherDetails
- Observe the result
![image](https://github.com/umbraco/Umbraco-CMS/assets/28688681/c336dded-0099-4af0-822c-debc5a57c4a5)

**Comment by truongtphat** [comment]
Actually I tried this code and it works as expected but following the documentation it doesn't.
![image](https://github.com/umbraco/Umbraco-CMS/assets/28688681/59766fd9-3d2f-40ad-aec2-939a4f6de567)

**Comment by truongtphat** [comment]
Hi @bergmania, 

Thanks for response.

I found the solution but if I follow the documentation and use `ExternalIndex` as the index name (default index) there aren't any searchers found.

**Comment by bergmania** [comment]
@truongtphat 

It is expected no searchers are returned when using `ExternalIndex`. Sorry, I can't see in the documentation how this was your expectation?

If there is some sentences you think would be better if refraced, feel free to make a PR for the docs

**Comment by abjerner** [comment]
If I may add something, the example with `ExamineManager.TryGetSearcher("ExternalSearcher", out var searcher)` is probably a bit outdated.

This used to be the way to get the searcher, but with newer Umbraco versions (probably 9+), there is no `ExternalSearcher`. Instead you have to get a reference to the `IIndex` first via `ExamineManager.TryGetIndex("ExternalIndex", out var index)`, and then get the searcher via it's `Searcher` property.

The example for getting the searcher is probably still valid for custom indexes, but not for getting the searcher if a specific index.

**Comment by markadrake** [comment]
I found myself here because [the Examine Manager doc has non-working code samples](https://docs.umbraco.com/umbraco-cms/reference/searching/examine/examine-manager). I'm not sure why this has been closed before updating the documentation. I'm just a FED, but the code sample clearly does not work and should be updated accordingly. 

<img width="784" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/713355/032bbbf9-3d7a-416b-845b-3a5bee779581">

This isn't clear, and this code does not work in a standard install of Umbraco 13.
